src/NuGet.Clients/NuGet.SolutionRestoreManager/RestoreOperationLogger.cs

```diff
@@ -63,13 +64,13 @@ internal sealed class RestoreOperationLogger : ILogger
 
         public async Task StartAsync(
             RestoreOperationSource operationSource,
-            ErrorListProvider errorListProvider,
+            Lazy<ErrorListTableDataSource> errorListDataSource,
```

Comment:
Running it here instead of during the MEF import seems safer considering the number of RPS regressions this area has had.